client/components/LearnerViewDirective.js

```diff
@@ -249,10 +249,20 @@ tie.directive('learnerView', [function() {
         $scope.submitCode = function(code) {
           SolutionHandlerService
             .processSolutionAsync(prompts[currentPromptIndex], code, language)
-            .then(appendFeedback);
+            .then(setFeedback);
         };
 
-        clearFeedback();
+        $scope.feedbackParagraphs = [
+          'Greetings!',
+          'This set of exercises focuses on string manipulation.',
```

Comment:
Per offline discussion, we ultimately want the intro text to be customizable and unique to each question set. It's fine to leave this as a TODO for this commit and address in a subsequent commit, ideally for Alpha release.